ember-resources/src/function-based/types.ts

```diff
@@ -10,7 +10,7 @@ export interface InternalFunctionResourceConfig<Value = unknown> {
   [INTERNAL]: true;
 }
 
-export const CURRENT = Symbol('ember-resources::CURRENT');
+export const CURRENT = Symbol('ember-resources::CURRENT') as unknown as 'CURRENT';
```

Comment:
this is the fix: typescript has been annoying with symbols and inferred types lately. :(